### DISCUSSION TITLE
blog/write-blog-on-white-paper

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#5] [Announcements]
# blog/write-blog-on-white-paper

现在，许多人像规划企业网站那样的规划个人博客：功能要齐全，界面要好看，经营要持续、迁移要方便，交流要方便。写作要坚持是最后考虑的一条。

https://rercel.com/blog/write-blog-on-white-paper

**Comment by ghost** [comment]
人们很难分得清商业驱动站点(wordpress)/兴趣爱好(随处都能写)的区别，他们把兴趣当作一门生意去看待，但从搜索引擎的角度来看，博客的没落是现实的。

> “等我建立一个完美的博客，就可以好好写文章了”。然而，你将永远都不能等到这一天。

我曾也想过这个问题，一个博客究竟需要怎样的功能，很多时候是受到别人吹嘘一些主题的影响，但我现在觉得sqlite都不想折腾了，需要的是**现在 立刻 就写**，正如在这个markdown评论框一样（好像是giscus），内容能传达给读者才是最重要的。

> 他们中的极个别人出于极为偶然的原因来到了这里，然后给了我们几秒时间浏览下小标题，然后永远消失在互联网的海洋中。

而且现在人们对长文/博客都失去了兴趣，标题不过关几乎点击率就少，不说博客能运行多久，希望能保持阅读和写作的习惯。

**Reply by Pathsis** [reply]
> 现在人们对长文/博客都失去了兴趣，标题不过关几乎点击率就少，不说博客能运行多久，希望能保持阅读和写作的习惯.

很早以前，我对自己写博客就有一个愿望，“长话短说”，要点式写作，或者用某个词，叫做“格式化”你的博文。尽管人们对长文失去了耐心，但一篇精炼的文章还是可以吸引人们花费更多的时间阅读。

我自己对博客写作的一个想法是，它在一定程度上反映了作者的写作能力和风格，文章清晰、直白、文气通顺，是特别重要的目标，我时常将这种目标与自己的论文写作联系在一起。

**Reply by ghost** [reply]
感谢你的回复，

> 我自己对博客写作的一个想法是，它在一定程度上反映了作者的写作能力和风格，文章清晰、直白、文气通顺，是特别重要的目标，我时常将这种目标与自己的论文写作联系在一起。

很有趣的观点，我经常查阅文献发现一个问题，大部分文献篇目很长但内容很少，以至于我常只看abstract和conclusion来了解大概，但一些著作值会很吸引人，值得从头到尾细读，确实能反映作者本人的写作能力。

> 在白纸上写日志的年代，没什么人会如此纠结写作条件。但是在今天，却有那么多人纠结自己博客的“百年大计”？

突然想到另一个例子，Ipad的“买前生产力 买后爱奇艺”问题，苹果公司的营销总是强调产品的**外在价值**，一种物超所值的错觉；回到博客，它的内在价值只是内容的载体，在互联网它是web1.0，单向的内容提供，内在价值是冰冷的HTMLs解析渲染到客户端，外在价值是各种交互也就是web2.0，当然现在的”web3“更是脱离web+内容本身的价值，所以折腾博客确实是有需要考虑的”外在价值“。

也有一个反例，是我在笔记本上阅读电子书发现的，鼠标滑轮要滑很多次要翻页，但不能快捷键翻页因为pdf的尺寸和屏幕不兼容无法展示单页内容，但一个平板或kindle是一种助推，行为心理学的实践，因为书更容易直接打开和阅读方便，才导致阅读频率的上涨；但我不太能确定博腾博客的事项是否存在行为经济学上的正反馈，但好的UI/UX设计可能促进用户的选择。

**Reply by Pathsis** [reply]
不清楚你的专业领域是什么。我做法律哲学研究的，平时主要还是阅读经典文本，这些文本会逐字逐句反复阅读，至于二手文献和论文，也是一再精选，许多文章很好，以至于在读后忍不住将其翻译过来。所以，对我来说，那些只需要通过摘要或结论就能把握的论文，我阅读很少。

我因为做英美法哲学的，向外扩展做了一些英美哲学的研究。当代英美哲学写作一向注重清晰，可以说清晰是作者的最高追求之一。在我学习英美哲学之处，也将思想与写作清晰铭记在心，在学习的过程中，对这个问题做了许多的思考，也把一些思考写了下来。

为什么当代英美哲学如此强调清晰？原因有许多，其中最重要的一个原因是，英美哲学十分注重逻辑研究、论证，并采取经验性和日常性的用语，研究者对发明概念没有兴趣，而是喜欢用已有的日常概念来说明问题。

与之相比，当代欧陆哲学，特别是当代法国哲学，特别喜欢发明概念，用这些概念来阐释日常经验，试图带给人理论上的“惊讶”，甚至意图解构或戏谑人们的日常经验。

这可以从初学者面对这两种风格的哲学的一般感受窥得一二。初学者常会被当代法国哲学著作搞得晕头转向，迷失在一堆新词和概念，以及错综复杂的隐喻之中。与之相比，当代英美哲学的语言就特别直白，甚至可以说有点憎恶“隐喻”和任何拟人化的语言。许多初学者第一次阅读当代英美哲学，很可能惊讶于一个事实，原来哲学也可以这么清晰直白。

至于那些熟悉了当代英美哲学的人，常常对当代法国哲学的叙事和论证几乎难以忍受。据说美国大学很多不承认法国哲学博士学位，不知道传言是否为真。我自己蹭感慨：当代法国哲学是法兰西民族的行为艺术。

当然，我这里没有褒贬的意思，它们是两种带有不同风格和旨趣的哲学，而且，当代法国哲学之所以这样做，有其深刻的理由，其中一个大概是，哲学和文学的界限不如一般人所想的那样清晰，逻辑在隐喻之下。美国实用主义哲学家罗蒂对当代英美法哲学的“叛变”，可以作为一个案例来研究。另外，当代英美哲学的大宗师维特特根斯坦的哲学观念，在很大程度上支持这种哲学向文学的复归，不过这里我就不多谈了。

好像扯远了。不过，我所以谈到这个问题，也是想到似乎你受到当代欧陆哲学的影响更大一些，这从你的文章中可以看到。你的写作无疑也受到了欧陆哲学风格的影响。

回到我们自己的写作本身。除了上面讲的大问题之外，我自己在写作中会特别注意使用短句，特别注意是否文气顺畅，我觉得这特别重要。对于博客写作，我经常许下愿望，每一篇文章字数都不超过2500，最好在1000字以内，也许只有这样硬性的规定，才能迫使我挤掉不必要的水分和冗余。

最后，对于你说而博客UI/UX的外在价值的问题，我想，这个问题所以发生，大概只是因为作为博客作者的我们，有太多的自由，而自由导致了无尽的烦恼。我始终认为从一个读者的角度来看，博客的UI/UX几乎是没有存在感的。当然，我们自己折腾博客，在很大程度上是将其作为某种游戏来玩耍的。这又是另一个问题了。
